<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="energy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string>Energy </string>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;8514oem&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 14, 90);
font: 700 40pt &quot;8514oem&quot;;
color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="health">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string>Energy </string>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;8514oem&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="health_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 40pt &quot;8514oem&quot;;
color: rgb(85, 255, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Health</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font:1pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>Mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font:1pt &quot;8514oem&quot;;</string>
    </property>
    <property name="text">
     <string>Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemies_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);

</string>
    </property>
    <property name="text">
     <string>Number of 
enemies</string>
    </property>
   </widget>
   <widget class="QLabel" name="health_">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
</string>
    </property>
    <property name="text">
     <string>Number of 
health packs</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>118</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font:1pt &quot;8514oem&quot;;
</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Stop Game</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>499</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type commad </string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom in/out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="game_menu">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="end_game"/>
    <addaction name="start_game"/>
    <addaction name="rerun_game"/>
    <addaction name="save_game"/>
   </widget>
   <widget class="QMenu" name="mode_menu">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="manual"/>
    <addaction name="automatic"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="text_mode"/>
    <addaction name="graphical_mode"/>
    <addaction name="colour_mode"/>
   </widget>
   <addaction name="game_menu"/>
   <addaction name="mode_menu"/>
   <addaction name="view_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="end_game">
   <property name="text">
    <string>End Game</string>
   </property>
  </action>
  <action name="start_game">
   <property name="text">
    <string>Start Game</string>
   </property>
  </action>
  <action name="rerun_game">
   <property name="text">
    <string>Re-run Game</string>
   </property>
  </action>
  <action name="manual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="automatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="save_game">
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="text_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="graphical_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graphical</string>
   </property>
  </action>
  <action name="colour_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colours</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
